### PR TITLE
Improve Rust CI performance (`dev/graph`)

### PR DESCRIPTION
**Pull request by TimDiekmann**
See #779

### CODE REVIEW

File: packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs [hunk 1]
```diff
@@ -1,7 +1,6 @@
 //! The Axum webserver for accessing the Graph API datastore operations.
 //!
-//! Handler methods are grouped by routes that make up the `web_api`. See [`data_type`] for a
-//! grouped router example
+//! Handler methods are grouped by routes that make up the `web_api`.
```
Comment (Alfred-Mountfield) [inline packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs:3]: Oops can we drive-by this. It's not the `web_api` anymore but actually the REST one.

Comment (TimDiekmann) [reply packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs:3]:
```suggestion
//! Handler methods are grouped by routes that make up the REST API.
```

**Review by Alfred-Mountfield — APPROVED** [review]
Here's to having a faster "CI garph" (branch name 😁) 🎉 🚀

**Review by Alfred-Mountfield — APPROVED** [review]
🚀

**Review by Alfred-Mountfield — APPROVED** [review]

**Review by thehabbos007 — APPROVED** [review]
